CONTRIBUTING.md

```diff
@@ -0,0 +1,78 @@
+
+# Contributing to rsfbclient
+
+Currently this github repository is the main place for official discussion (via issues and pull requests)
+
+## Making and suggesting changes
+
+### Pull request or issue?
+* For small improvements or obvious bug fixes to code or documentation you can just make a pull request.
+ * For larger changes/suggestions that modify the API or bigger architectural changes, open an issue for discussion and feedback. 
+ * If you believe something should be changed in the crate's meta information (documentation, README.md, CONTRIBUTING.md, etc) you can create an issue for that. If appropriate, please provide links to existing resources that you feel may be useful.
+* Updates to out-of-date information, and small changes to existing files can be submitted directly as a PR. 
+
+### In all cases
+* Please read the section about Github Workflow. It is important that submitted PRs pass our automated tests and are correctly formatted.
+* If you are not sure about something, please check existing documentation before creating a PR or issue.  Since this crate is still fairly young, it is considerate, but not required, to also search for information that has already been discussed in old PRs and issues.
+* Note that this repository holds the source for several crates. If proposed changes are very drastic, it may be appropriate to implement it in a separate crate and/or put it behind a feature gate.
+* You may be asked to write or modify some tests after submitting an issue or PR. We will help with this where we can, if needed. See the section on testing for more information.
+
+### A template for commit messages
+(TODO)
+
+## Github workflow
+This repository uses Github actions. Upon submitting a pull request, it will automatically:
+* Check that code formatting in the PR commit agrees with the output of `cargo fmt --all`
+* Run all tests  from rust modules declared at the bottom of `src/tests/mod.rs`. These tests are run for many different combinations of platform and Firebird configurations.
+
+You can (and should) run these tests for your configuration using the command `cargo test -- --test-threads 1` 
+
+See the section on testing for more information.
+
+## Testing
+### Setting up a test database `test.fdb`
+#### Using the docker image
+  TODO
+#### Manually
+  * To set up a Firebird database for testing you will need to install the Firebird server. This process may vary by platform. (TODO: notes for different platforms/server versions)
```

Comment:
We can refer to the [official documentation](https://www.firebirdsql.org/file/documentation/html/en/firebirddocs/qsg3/firebird-3-quickstartguide.html#qsg3-installing-server)